package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "npm run build && npm-run-all --parallel watch-other watch-js watch-scss watch-i18n server",
     "server": "http-server -c-1 dist",
     "copy-html": "mkdir -p dist && cp src/*.html dist/",
-    "build": "npm run copy-html && npm run build-all-css && npm run build-js && npm run build-i18n",
+    "build": "npm run copy-html && npm run build-i18n && npm run build-all-css && npm run build-js",
```

Comment:
Why the change of ordering?

Reply:
Because `build-i18n` build `i18n.js` file in src directory which is then used by `build-js` script